Justfile

```diff
@@ -207,9 +207,9 @@ test-generative *ARGS: devenv
     #!/usr/bin/env bash
     set -euo pipefail
 
-    examples=${GENTEST_EXAMPLES:-200}
+    examples=${GENTEST_EXAMPLES:-100}
     [[ -v CI ]] && echo "::group::Run tests (click to view)" || echo "Run tests"
-    GENTEST_EXAMPLES=$examples GENTEST_COMPREHENSIVE=t $BIN/python -m pytest \
+    GENTEST_EXAMPLES=$examples $BIN/python -m pytest \
```

Comment:
I think we are going to need something that does the job of the comprehensive check, otherwise it's too easy to forget to add new operations to the generative tests. But I've had an idea of how to do this which might dramatically speed things up, and let us get away with far fewer examples in CI. I need to do some experimenting first.

Reply:
yes, turning it off here was just throwing it over the fence to #1242